tasks/sanity-inspect-image.yaml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: sanity-inspect-image
+spec:
+  params:
+    - name: SOURCE_IMAGE
+      description: the fully qualified image name
+      default: quay.io/hongweiliu/skopeo-inspect-test:latest
+      type: string
+    - name: BASE
+      description: is to get base image?
+      default: 'false'
+      type: string
+  steps:
+  - name: inspect-image
+    image: quay.io/redhat-appstudio/hacbs-test:latest
+    script: |
+      if [ $(params.BASE) == 'true' ];then
+        echo "Inspecting base image for source image: $(params.SOURCE_IMAGE)"
+        if [ -f "$(workspaces.conftest-ws.path)/base_image_inspect.json" ];then
+          echo "$(workspaces.conftest-ws.path)/base_image_inspect.json exists"
+        else
+          BASE_IMAGE="$(cat $(workspaces.conftest-ws.path)/image_inspect.json | jq -r ".Labels.\"org.opencontainers.image.base.name\"")"
```

Comment:
`BASE_IMAGE="$(jq -r ".Labels.\"org.opencontainers.image.base.name\"" $(workspaces.conftest-ws.path)/image_inspect.json"` -> no need to use `cat`

Reply:
Thank you!